test/controllers/mobile_app/sensors_controller_test.rb

```diff
@@ -11,22 +11,24 @@ class SensorsControllerTest < ActionDispatch::IntegrationTest
     end
 
     test 'should get index with latest measurement' do
-      @sensor.measurements << create(:measurement, temperature: 20)
+      @sensor.measurements << create(:measurement, temperature: 20, created_at: DateTime.parse('2020-08-08'))
 
       get mobile_app_sensors_url, env: public_auth_header
 
       assert_response :success
       assert_equal(parsed_response.first['latest_temperature'], 20)
+      assert_equal(parsed_response.first['latest_sensor_update'], DateTime.parse('2020-08-08').to_time.to_i)
```

Comment:
You're repeating the implementation with `to_i`. I guess that's fine in this case.

Reply:
Yeah, I wondered as well whether I should inline the timestamp. Might actually be the better idea, I'll change it.